test/README.md

```diff
@@ -155,6 +155,26 @@ import pdb; pdb.set_trace()
 anywhere in the test. You will then be able to inspect variables, as well as
 call methods that interact with the bitcoind nodes-under-test.
 
+If further introspection of the bitcoind instances themselves becomes
+necessary, this can be accomplished by first setting a pdb breakpoint
+at an appropriate location, running the test to that point, then using
+`gdb` to attach to the process and debug:
+
+```bash
```

Comment:
Yeah - if gdb can't attach to a specific PID, the error message is actually buried in the middle of the gdb text so it's not immediately obvious what the problem is. I think including the warning is helpful here.

Also, my above invocation was wrong:

```bash
cat /tmp/user/1000/testo9vsdjo3/node0/regtest/bitcoind.pid
gdb /home/example/bitcoind !$
```

it should be:

```
cat /tmp/user/1000/testo9vsdjo3/node0/regtest/bitcoind.pid
gdb /home/example/bitcoind <pid>
```

(sorry - trying to be too clever!)